src/component/header/index.tsx

```diff
@@ -45,8 +47,10 @@ export default function Header(props: HeaderProps) {
         ref: dropdownRootRef,
       }}
       button={{
-        wrap(node) {
-          return <Link to={basePath + "/members"}>{node}</Link>;
```

Comment:
I tend to prefer wrapping with `Link` because it means you can see where it will take you before you click on it (in the status bar). Also you can right click and select "Open in a new tab". What do you think?

Reply:
I see what you mean, I'll follow what you did in your previous PR then. Thanks